tests/vmcheck/test-layering-scripts.sh

```diff
@@ -77,6 +77,25 @@ vm_cmd cat /usr/lib/prefixtest.txt > prefixtest.txt
 assert_file_has_content prefixtest.txt "/usr"
 echo "ok script expansion"
 
+vm_rpmostree rollback
+vm_reboot
+vm_rpmostree cleanup -p
+vm_build_rpm scriptpkg4 \
+             transfiletriggerin "/usr/bin" 'cat >/usr/share/transfiletriggerin-bin.txt'
+vm_build_rpm scriptpkg5 \
+             transfiletriggerin "/usr/lib" 'cat >/usr/share/transfiletriggerin-lib.txt'
+vm_build_rpm scriptpkg6
+vm_rpmostree pkg-add scriptpkg{4,5}
+vm_rpmostree ex livefs
+vm_cmd cat /usr/share/transfiletriggerin-bin.txt > transfiletriggerin-bin.txt
+# Test that we actually got new files
+assert_file_has_content transfiletriggerin-bin.txt /usr/bin/scriptpkg5
+# And a wholesale find comparison
+vm_cmd cat /usr/share/transfiletriggerin-lib.txt | sort > transfiletriggerin-lib.txt
+vm_cmd find /usr/lib | sort > usrlib.txt
+diff -u usrlib.txt transfiletriggerin-lib.txt
+echo "ok transfiletriggerin"
```

Comment:
How about also testing the following scenarios:

- pattern that doesn't match anything
- pkg with multiple `%transfiletriggerin` -- as opposed to `%post`, multiple file-based triggers are supported (most likely because it makes sense to split them by pattern)